docs/uml-class-diagrams/database/test/DatabaseGisInterfaceImplementationTests/DatabaseGisInterfaceImplementationTests.puml

```diff
@@ -2,7 +2,10 @@
 
 package edu.oakland.test.database {
     class DatabaseGisInterfaceImplementationTests {
-
+        + void dataPointInIsDataPointOut()
+        + void modeInIsModeOut()
+        + void currentSatMatchNextSat()
```

Comment:
Minor nitpick, but can we call this "Gets" instead of "Matches", because the goal is the the one we put in is NOT the same one we get back back out.
```suggestion
        + void currentSatGetsNextSat()
```